Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -2102,19 +2103,28 @@ static Ref<CSSValue> altTextToCSSValue(const RenderStyle& style)
     return CSSPrimitiveValue::create(style.contentAltText());
 }
 
-static Ref<CSSValueList> contentToCSSValue(const RenderStyle& style)
+static Ref<CSSValue> contentToCSSValue(const RenderStyle& style)
 {
     CSSValueListBuilder list;
     for (auto* contentData = style.contentData(); contentData; contentData = contentData->next()) {
-        if (auto* counterContentData = dynamicDowncast<CounterContentData>(*contentData))
-            list.append(CSSPrimitiveValue::createCounterName(counterContentData->counter().identifier()));
-        else if (auto* imageContentData = dynamicDowncast<ImageContentData>(*contentData))
+        if (auto* counterContentData = dynamicDowncast<CounterContentData>(*contentData)) {
+            RefPtr counterStyle = CSSPrimitiveValue::createCustomIdent(counterContentData->counter().listStyleType().identifier);
```

Comment:
`auto`?

Reply:
`auto` will give `Ref` instead of `RefPtr` which is what is expected here.